.github/ISSUE_TEMPLATE/parameter-testing.yml

```
@@ -0,0 +1,38 @@
+name: Parameter Testing
+description: Template for tracking parameter testing issues
+title: "[Parameter Testing] "
+labels: "Parameter-testing"
+assignees: [optional-assignee]
+body:
+  - type: markdown
+    attributes: 
+      value: |
+        Please fill out the sections below to clearly and concisely describe: 
+        1. the parameter(s) tested,
+        2. the exact version of the test tool (Expts_manager) used.
+
+  - type: textarea
+    attributes:
+      label: Parameter Tests Description
+      placeholder: "including a short description and Parameter Name(s): Value(s) Tested"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Expts_manager Version
+      placeholder: Git URL or commit hash used
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: YAML input file
+      placeholder: Copy the yaml input file used
```

Can you include an empty code block as the placeholder. E.g. something likt the following?
```suggestion
      placeholder: 
        ```yaml
        Copy the yaml input file used
        ```
```